deploy/gke/redaction/otel-config.yaml

```diff
@@ -21,8 +21,6 @@ receivers:
 exporters:
   # The googlecloud exporter sends traces to google cloud trace.
   googlecloud:
-    retry_on_failure:
```

Comment:
Is removing this part of the fix or is it just removed since it is not necessary ?

Reply:
The new collector version no longer supports that configuration, and the default is now false.  See https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/25900